src/lib/bootloader/auto_client.rb

```diff
@@ -74,7 +76,16 @@ def change
     #
     # return map or list
     def export
-      Export2AI(Yast::Bootloader.Export)
+      # it is needed to have information about storage configuration to understand current config
+      Yast::BootStorage.detect_disks
+
+      config = BootloaderFactory.current
+      config.read if !config.read? && !config.proposed?
```

Comment:
well, it is my internal rule to use `unless` only if there is single parameter, this way it is more like `if !(config.read? || config.proposed?)` is need at least for me more brain capacity to read it :)